feature-libs/pdf-invoices/karma.conf.js

```diff
@@ -4,9 +4,8 @@
 module.exports = function (config) {
   config.set({
     basePath: '',
-    frameworks: ['parallel', 'jasmine', '@angular-devkit/build-angular'],
```

Comment:
[QUESTION]

Why is removing `karma-parallel` related to the upgrade of rxjs?

btw. I can understand that for small libraries that have little unit tests, the parallelization of the unit tests execution might not bring much value. `pdf-invoices` might be one of such libraries. But I'm wondering why you made this change in _this_ rxjs-upgrad PR ?